documents/resources/files/config.py

```diff
@@ -29,6 +32,15 @@ def response_handlers(self):
             **entrypoint_response_handlers,
         }
 
+    error_handlers = {
```

Comment:
Please move this dictionary to common and just reference here. We will later on add an option to oarepo-model-builder to do the referencing